run/markdown-preview/renderer/main.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, request
+import markdown
+import os
+
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["POST"])
+def index():
+    data = request.get_data(as_text=True)
+    return markdown.markdown(data)
```

Comment:
nit: comment what `markdown.markdown()` is doing.

Reply:
Added comment